routes/disaggregation.py

```diff
@@ -0,0 +1,91 @@
+import os
+from datetime import datetime, timedelta
+import logging
+from flask import Blueprint, jsonify, request
+from flask import current_app as app
+from discovergy.discovergy import Discovergy
+
+
+logger = logging.getLogger(__name__)
+IndividualDisaggregation = Blueprint('IndividualDisaggregation', __name__)
+GroupDisaggregation = Blueprint('GroupDisaggregation', __name__)
+
+
+def read_parameters():
+    """ Use the given parameters. """
+    start = round((datetime.now() - timedelta(hours=48)).timestamp() * 1e3)
```

Comment:
What is 1e3? And Why?

Reply:
What you get by timestamp() is the seconds precision, but the Discovergy API requires milliseconds. * 1e3 means ^(-3), so you get the ms precision and round it afterwards to have an integer value as requested by their API. I clarified this with a comment.